.github/workflows/build-test-docker.yml

```diff
@@ -1,4 +1,4 @@
-name: build-test
+name: GitHub-ci
```

Comment:
Not a fan of the CamelCase-with-hyphen. How about call it `github-ci` and rename the yaml file `github-ci.yml`

Reply:
done!